urlsearchparams/page.css

```diff
@@ -0,0 +1,26 @@
+body {
```

Comment:
I'd recommend ditching the custom styles for HTML elements, for the reasons mentioned above re: the custom font. The class styles for layout are fine, obviously.